.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
ests\/database\/factories\/OrderFactory.php":1919937318,"tests\/database\/factories\/OrderItemFactory.php":982154050,"tests\/database\/factories\/RedeemedCouponFactory.php":3122826970,"tests\/database\/factories\/SubscriptionFactory.php":636266896,"tests\/database\/factories\/RefundFactory.php":2029386449,"tests\/Http\/Controllers\/WebhookControllerTest.php":1123044535,"tests\/FirstPayment\/Actions\/AddGenericOrderItemTest.php":677532673,"tests\/FirstPayment\/Actions\/StartSubscriptionTest.php":341072480,"tests\/FirstPayment\/Actions\/ApplySubscriptionCouponToPaymentTest.php":764212356,"tests\/FirstPayment\/Actions\/AddBalanceTest.php":2210883002,"tests\/FirstPayment\/Actions\/StartSubscriptionWithPlanIntervalArrayTest.php":261567694,"tests\/FirstPayment\/FirstPaymentBuilderTest.php":3414240735,"tests\/FirstPayment\/FirstPaymentHandlerTest.php":535943095,"tests\/UpdatePaymentMethod\/UpdatePaymentMethodWithoutAddingToBalanceTest.php":3467045743,"tests\/UpdatePaymentMethod\/UpdatePaymentMethodTest.php":3173747804,"tests\/Order\/FakeOrderItemPreprocessor.php":1830039631,"tests\/Order\/OrderNumberGeneratorTest.php":257079643,"tests\/Order\/OrderCollectionTest.php":3763781900,"tests\/Order\/OrderInvoiceSubscriberTest.php":972179658,"tests\/Order\/OrderItemCollectionTest.php":1808984306,"tests\/Order\/OrderItemTest.php":2394075327,"tests\/Order\/InvoiceTest.php":4044797481,"tests\/Order\/OrderItemPreprocessorCollectionTest.php":63191292,"tests\/Order\/OrderTest.php":372760353,"tests\/SubscriptionBuilder\/MandatedSubscriptionBuilderTest.php":492561035,"tests\/SubscriptionBuilder\/FirstPaymentSubscriptionBuilderApplyCorrectTaxTest.php":1844091601,"tests\/SubscriptionBuilder\/InvalidatingCouponHandler.php":3797921479,"tests\/SubscriptionBuilder\/FirstPaymentSubscriptionBuilderTest.php":909629863,"tests\/Plan\/ConfigPlanTest.php":1390790695,"tests\/Plan\/ConfigPlanRepositoryTest.php":2731909415,"tests\/Helpers\/HelpersTest.php":21708501,"tests\/EventServiceProviderTest.php":901459568,"tests\/SwapSubscriptionPlanTest.php":3922404503,"tests\/CashierTest.php":1079069368,"tests\/SubscriptionTest.php":494395828,"tests\/BillableTest.php":1455610985,"tests\/BaseTestCase.php":760081172,"tests\/Mollie\/UpdateMolliePaymentTest.php":2278010199,"tests\/Mollie\/BaseMollieInteractionTest.php":4227849375,"tests\/Mollie\/GetMollieMandateTest.php":84204709,"tests\/Mollie\/CreateMollieCustomerTest.php":1350788803,"tests\/Mollie\/GetMollieRefundTest.php":3224566903,"tests\/Mollie\/GetMollieCustomerTest.php":206342844,"tests\/Mollie\/GetMolliePaymentTest.php":2751900370,"tests\/Mollie\/CreateMolliePaymentTest.php":3706303709,"tests\/Mollie\/CreateMollieRefundTest.php":3545454736,"tests\/CashierServiceProviderTest.php":3627601961,"tests\/Credit\/CreditTest.php":3564313542,"tests\/Coupon\/RedeemedCouponTest.php":2494995236,"tests\/Coupon\/CouponOrderItemPreprocessorTest.php":251911886,"tests\/Coupon\/ConfigCouponRepositoryTest.php":3733913546,"routes\/webhooks.php":1754211312}}
```

Comment:
Can you remove this file? Add it to .gitignore if necessary

Reply:
Done